javaSE/src/main/java/com/smartdevicelink/managers/lifecycle/LifecycleManager.java

```diff
@@ -642,6 +642,33 @@ private boolean removeOnRPCRequestListener(FunctionID requestId, OnRPCRequestLis
 
 
     private void sendRPCMessagePrivate(RPCMessage message){
+        //FIXME this is temporary until the next major release of the library where OK is removed
+        if (message.getMessageType().equals(RPCMessage.KEY_REQUEST)) {
+            RPCRequest request = (RPCRequest) message;
+            if(FunctionID.SUBSCRIBE_BUTTON.toString().equals(request.getFunctionName())
+                    || FunctionID.UNSUBSCRIBE_BUTTON.toString().equals(request.getFunctionName())
+                    || FunctionID.BUTTON_PRESS.toString().equals(request.getFunctionName())) {
+
+                ButtonName buttonName = (ButtonName) request.getObject(ButtonName.class, SubscribeButton.KEY_BUTTON_NAME);
+
+                if (rpcSpecVersion != null && rpcSpecVersion.getMajor() < 5) {
+
+                    if (ButtonName.PLAY_PAUSE.equals(buttonName)) {
+                        request.setParameters(SubscribeButton.KEY_BUTTON_NAME, ButtonName.OK);
+                    }
+                } else { //Newer than version 5.0.0
+                    if (ButtonName.OK.equals(buttonName)) {
+                        RPCRequest request2 = new RPCRequest(request);
+                        request2.setParameters(SubscribeButton.KEY_BUTTON_NAME, ButtonName.PLAY_PAUSE);
+                        request2.setOnRPCResponseListener(request.getOnRPCResponseListener());
+                        sendRPCMessagePrivate(request2);
+                        return;
+                    }
+                }
+            }
+        }
+
```

Comment:
Same comments as above regarding try/catch block and second false positive on the else block